app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+from random import randint
+
+
+def flip_coin() -> dict:
+    res = []
+    for i in range(10000):
```

Comment:
Consider using a more descriptive variable name instead of `res`. For example, `results` or `distribution`. 🤔